### ISSUE TITLE
Reading age research plan

### ISSUE THREAD

**Issue by amyj2110** [nhsuk/nhsuk-service-manual#939]
## What

Create a plan to conduct research about reading age.

## Why

- To understand what there is to do around reading age research.

## Must do
- [ ] Agree research scope with @sarawilcox 
- [ ] Create research plan
- [ ] Feed back with @sarawilcox and agree next steps

**Comment by cjforms** [comment]
Please also include research around functional literacy. 

I think I've already mentioned that the concept of reading age was discredited many years ago - but I'm not sure I've explicitly mentioned the concept that replaced it. For example, the OECD defines literacy in terms of levels. There's a good summary here:
https://literacytrust.org.uk/parents-and-families/adult-literacy/what-do-adult-literacy-levels-mean/

**Comment by sarawilcox** [comment]
Thanks @cjforms. 

I agree. [Reading age](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/258) is a problematic concept and we should probably be giving people guidance about functional literacy levels instead. For example, we know from the Literacy Trust that 1 in 7 adults in England have literacy levels at or below Entry Level 3, which is equivalent to the literacy skills expected of a 9 to 11-year-old. Adults with skills below Entry Level 3 may not be able to understand price labels on pre-packaged food or understand household bills.

However, content designers seem to find our current guidance on a reading age of 9 to 11 years old helpful, both when dealing with stakeholders and also to help them simple, clear content. 

We want to find out more about what content designers need to know about this in order to write simply and clearly.

**Comment by cjforms** [comment]
That's a cracking challenge. Sadly, I agree that we don't yet have a simple way to express the concept of functional literacy in a way that makes it easy for a content designer to use in a meeting with a stakeholder. 

I'm really interested in the results of your research.

**Comment by henocookie** [comment]
Some useful context of why this research is being planned is available on the [Reading age service manual backlog issue](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/258)

**Comment by sarawilcox** [comment]
@cjforms I'm now wondering if we should consider not just functional literacy skills but also numeracy and digital skills. Any or all of these may come into play in online health content.

**Comment by cjforms** [comment]
Yes, the health literacy page makes it clear that numeracy is important. 

A bit tangential to this issue, but related to the health literacy page:
I can add a personal example: being sent home from hospital after serious illness with a large bag of drugs AND cognitive deficits (temporary disability) due to the effects of treatment. It took me several hours to create a list on paper that told me which medications to take when due to their various requirements ("take with food" / "take at least two hours after food or an hour before" / "take 3x day" / "take every 6 hours" and so on and on). 

Another example from that time: I was also sent home with a printed leaflet about how to call the hospital for additional help, and I absolutely could not find the number I needed despite several attempts. It was only several days later, after making a series of calls to wrong numbers and eventually tracking down the correct one, that I realised that the leaflet had a second side. 

Which makes me wonder about adding something on the health literacy page about how people with good literacy, numeracy, and digital skills may have that reduced by the effects of stress, illness, or treatment. 

Coming back to the research topic on this page: I can see that content designers and others may need a quick phrase or slogan that captures the ideas on the health literacy page to convey them in a meeting.  Ideally, I'm hoping that you might think about:
- a couple of eye-catching stickers with suitable slogans
- a set of posters that have slogans plus brief supporting text and longer references. 

I think the relevant supporting page in the service manual might be: "How to explain health literacy to colleagues". Or may that is the health literacy page, lightly edited. Only thinking aloud here.

**Comment by cjforms** [comment]
Coincidentally, I enjoyed this piece recently: https://medium.com/wehearthealthliteracy/to-be-clear-use-all-the-words-you-need-26ab469f38f6

**Comment by amyj2110** [comment]
Paused, for now.

**Comment by sarawilcox** [comment]
We have an action from the July Style Council meeting to do some UR around this. Also following a meeting with the Patient Information Forum about reading age today. There is clearly a sense among NHS.UK content designer and PIF members that some guidance on a reading age to aim for is helpful, with caveats and more guidance on what makes good content. But we need a better understanding of what content designers need and why.

I've picked up Henry's draft research plan and will discuss with Amy on Friday.

**Comment by sarawilcox** [comment]
Recommendation that we collaborate on the survey with:
- GOV.UK
- PIF
- Test and Trace
